src/Validation/Validator.php

```diff
@@ -619,9 +711,10 @@ public function lengthBetween($field, array $range, $message = null, $when = nul
             throw new InvalidArgumentException('The $range argument requires 2 numbers');
         }
         $extra = array_filter(['on' => $when, 'message' => $message]);
+
         return $this->add($field, 'lengthBetween', $extra + [
-            'rule' => ['lengthBetween', array_shift($range), array_shift($range)],
-        ]);
+                'rule' => ['lengthBetween', array_shift($range), array_shift($range)],
+            ]);
```

Comment:
Can you revert the autoformatting? It makes this change much larger/noisier than it needs to be.